src/pages/instances/StoragePoolMigrationTable.tsx

```diff
@@ -0,0 +1,82 @@
+import { FC } from "react";
+import { ActionButton, Button } from "@canonical/react-components";
+import { LxdInstance } from "types/instance";
+import { LxdDiskDevice } from "types/device";
+import { isRootDisk } from "util/instanceValidation";
+import { FormDevice } from "util/formDevices";
+import { LxdStoragePool } from "types/storage";
+import StoragePoolSelectTable from "./StoragePoolSelectTable";
+
+interface Props {
+  instance: LxdInstance;
+  onSelect: (pool: string) => void;
+  targetPool: string;
+  onCancel: () => void;
+  pools: LxdStoragePool[];
+  migrate: (pool: string) => void;
+}
+
+const StoragePoolMigrationTable: FC<Props> = ({
```

Comment:
```suggestion
const InstanceStoragePoolMigration: FC<Props> = ({
```
Not a table anymore, and I think we should be more specific to avoid the name clash with the volume pool migration.

Reply:
same goes for `ClusterMemberMigrationTable` -> `InstanceClusterMemberMigration`